lib/clone_kit/cloners/active_record_ruleset_cloner.rb

```diff
@@ -104,6 +102,22 @@ def initialize_cloner(operation)
       def connection
         ActiveRecord::Base.connection
       end
+
+      private
+
+      # This method has been deprecated in rails 5.2 (https://github.com/rails/rails/commit/2f45157f2ebbe4f3fa43d3998b459c8eb9ec2b89)
+      # Workarounds to this method started to fail other parts of the code base. Hence I am defining it here.
+      def arel_attributes_with_values_for_create(model)
```

Comment:
Exact same functionality of `arel_attributes_with_values_for_create` from [here](https://github.com/rails/rails/commit/2f45157f2ebbe4f3fa43d3998b459c8eb9ec2b89)